README.md

```diff
@@ -6,28 +6,52 @@ https://explorer.ooni.torproject.org/
 
 ## Setup
 
-### On Mac OS X
+### Docker
+
+We include a docker file.
```

Comment:
I suggest to publish it to [the hub](https://hub.docker.com/r/openobservatory/) as well, do you have any objections?

Reply:
Sure, go for it.